pants.toml

```diff
@@ -240,3 +241,6 @@ args = ["-Yrangepos", "-Xlint:unused"]
 
 [scala-infer]
 force_add_siblings_as_dependencies = false
+
+[pypi-audit]
+lockfile_vulnerability_excludes = { "python-default" = ["GHSA-w596-4wvx-j9j6"] }
```

Comment:
* Could you add a comment for why this can be ignored?  It's not just an example, correct?
* This is just me being unfamiliar with the space, but how do we end up with a GitHub advisory ID instead of a CVE or python specific one?

Reply:
Agree. I think every exception should be made with a note as to why it's being excluded. Perhaps this should be generally encouraged, so we could have a syntax like:
```suggestion
lockfile_vulnerability_excludes = { "python-default" = [{"id": "GHSA-w596-4wvx-j9j6", "note": "This is N/A for reasons a, b and c."}] }
```

and potentially warn/err if there is no `note`.

These excludes could then be _briefly_ mentioned, along with the note, when running `pants audit` for visibility.